docs/abi-spec.rst

```diff
@@ -326,16 +326,16 @@ An event description is a JSON object with fairly similar fields:
 
 - ``anonymous``: ``true`` if the event was declared as ``anonymous``.
 
-For example, 
+For example,
 
 ::
 
   pragma solidity ^0.4.0;
 
   contract Test {
     function Test(){ b = 0x12345678901234567890123456789012; }
-    event Event(uint indexed a, bytes32 b)
-    event Event2(uint indexed a, bytes32 b)
+    event Event(uint indexed a, bytes32 b);
+    event Event2(uint indexed a, bytes32 b);
```

Comment:
I wonder why this isn't caught be the tests. All the examples are compiled from the documentation.

Reply:
Ah, they are wrongly indented. Will fix that.